glean-core/android/src/test/java/mozilla/telemetry/glean/private/UuidMetricTypeTest.kt

```diff
@@ -46,12 +40,14 @@ class UuidMetricTypeTest {
             sendInPings = listOf("store1")
         )
 
+        // Check that there is no UUID recorded
+        assertFalse(uuidMetric.testHasValue())
+
         // Record two uuids of the same type, with a little delay.
-        val uuid = uuidMetric.generateAndSet()
+        uuidMetric.generateAndSet()
```

Comment:
Let's not change the test, unless it's required. By keeping the API the same, as suggested above, this could remain the same.